src/dmd/staticcond.d

```diff
@@ -96,6 +96,8 @@ bool evalStaticCondition(Scope* sc, Expression original, Expression e, out bool
 
         e = e.expressionSemantic(sc);
         e = resolveProperties(sc, e);
+        if (e)
```

Comment:
is this test necessary? its used unconditionally on [line 103](https://github.com/dlang/dmd/pull/10237/files#diff-d9f609df174a1d074cfc37b54558e504R103)

Reply:
I removed it. I'm a bit defensive while I'm still surprised by the abundant usage of null in dmd.